Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -984,6 +984,20 @@ CSSCounterStyleAtRulesEnabled:
     WebCore:
       default: true
 
+CSSCrossDocumentViewTransitionsEnabled:
+  type: bool
+  status: testable
+  category: css
+  humanReadableName: "CSS view-transitions cross-document support"
+  humanReadableDescription: "Enable support for CSS view-transitions cross-document"
+  defaultValue:
+    WebKitLegacy:
+      default: false
+    WebKit:
+      default: false
+    WebCore:
+      default: false
```

nit: maybe this, to be consistent with the VT flag, and because there's more than just CSS in this bucket:

```suggestion
CrossDocumentViewTransitionsEnabled:
  type: bool
  status: testable
  category: animation
  humanReadableName: "Cross-document View Transitions"
  humanReadableDescription: "Enable support for cross-document view transitions"
  defaultValue:
    WebKitLegacy:
      default: false
    WebKit:
      default: false
    WebCore:
      default: false
```